scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/compilers.yaml

```diff
@@ -0,0 +1,56 @@
+compilers:
```

Comment:
I modeled this after the `blueos` compiler specs.  Please let me know if we should modify the compiler list, e.g. for known compatibility issues.